src/mlpack/methods/ann/convolution_rules/naive_convolution.hpp

```diff
@@ -55,28 +55,32 @@ class NaiveConvolution
               const size_t dW = 1,
               const size_t dH = 1,
               const size_t dilationW = 1,
-              const size_t dilationH = 1)
+              const size_t dilationH = 1, const size_t appending = false)
```

Comment:
add appending option; when appending=true, the convolution just adds to the output instead of allocating a new matrix and then adding to it. Spares an allocation & some CPU time.

Reply:
I think ```appending``` should be a ```bool``` value then.

Reply:
sure, will change it to that then.